### PR TITLE
Feature/sign out function to page

### PR DESCRIPTION
**Pull request by gevans-ch**
Added sign out function to main search page on BADOS web

Unit tests are included. 
Has been tested locally on Docker Image 

Covers ticket: https://companieshouse.atlassian.net/browse/BI-6527

### CODE REVIEW

File: src/controller/bankrupt/bankrupt.controller.ts [hunk 1]
```diff
@@ -33,7 +34,7 @@ export const postSearchPage = async (req: Request, res: Response, next: NextFunc
     // Not found officers has to be rendered anyway with an empty list 
     if(!results.error || results.status === 404){
       const { itemsPerPage = 0, startIndex = 0, totalResults = 0, items = [] } = results.data || {};
-      return res.render('bankrupt', { itemsPerPage, startIndex, totalResults, items: formattingOfficersInfo(items), searched: true });
+      return res.render('bankrupt', { itemsPerPage, startIndex, totalResults, items: formattingOfficersInfo(items), searched: true , userEmail});
     } else {
       return res.status(results.status).render('error-pages/500');
     }
```
Comment (mouhajer-ch) [inline src/controller/bankrupt/bankrupt.controller.ts:40]: Retrieve the `userEmail = userSession.getLoggedInUserEmail(req.session); ` when is needed, inside the if condition, and remove spaces please (line 9 and 39). Thanks :)

File: src/views/layout.html [hunk 2]
```diff
@@ -45,6 +45,12 @@
   }) }}
 {% endblock %}
 
+{% block signOutBanner %}
+ 
+  {% include "components/sign-out-user-banner.njk" %}
+
```
Comment (mouhajer-ch) [inline src/views/layout.html:51]: Removing empty line on `signOutBanner`, it is much more readable

**Review by mouhajer-ch — COMMENTED** [review]
I would have added more test, for example an integration test that checks the presents of the signet logic. Like 
`
  describe('Should hit SCOTTISH_BANKRUPT_OFFICER Endpoint and return signout info', () => {
    it("should find " + SCOTTISH_BANKRUPT_OFFICER + " page url", async () => {
      return await request(app)
        .get(SCOTTISH_BANKRUPT_OFFICER)
        .set('Cookie', signedInCookie)
        .then(response => {
          expect(response.text).to.include('userWithPermission@ch.gov.uk');
          expect(response.text).to.include('/signout');
          expect(response.status).equal(200);
        });
    });
  });
`

File: src/controller/bankrupt/bankrupt-officer.controller.ts [hunk 3]
```diff
@@ -5,10 +5,10 @@ import { logger, formattingOfficersInfo } from '../../utils';
...
-
-    if(!bankruptOfficer.error){
+    
+    if (!bankruptOfficer.error) {
       const officer = (bankruptOfficer.data) ? formattingOfficersInfo([bankruptOfficer.data])[0] : {};
-      return res.render('bankrupt_officer', { bankruptOfficer: officer });
+      return res.render('bankrupt_officer', { bankruptOfficer: officer } );
     } else if( bankruptOfficer.status === 500) {
       return res.status(bankruptOfficer.status).render('error-pages/500');
     } else {
```
Comment (mouhajer-ch) [inline src/controller/bankrupt/bankrupt-officer.controller.ts:14]: I would not have committed a file just with spaces changes :)

Comment (gevans-ch) [reply src/controller/bankrupt/bankrupt-officer.controller.ts:14]: this was trying to revert previous changes to match main, mainly to avoid merge issues

**Review by mouhajer-ch — APPROVED** [review]
It looks good!

**Pull request merged**